source/containers.tex

```diff
@@ -5947,7 +5947,6 @@
 
 \begin{itemdescr}
 \pnum
-\pnum
```

Comment:
As https://github.com/cplusplus/draft/issues/3982#issuecomment-625452674 points out, this should be moved to the following `\expects` paragraph.

Reply:
Done.